resources/views/pages/contract/index.blade.php

```diff
@@ -10,11 +10,11 @@
 	<section>
       <div class="container">
       {{ Breadcrumbs::render('contractors') }}
-        <h1 class="ws-10 font-weight-bold">Contracted Companies and Organisations</h1>
+        <h1 class="ws-10 font-weight-bold">Contracted contractors and Organisations</h1>
```

Comment:
I think this should be "Companies", "Contracted contractors" sounds like repetition.

Reply:
Noted